src/pip/_internal/wheel.py

```diff
@@ -733,10 +733,12 @@ def _build_one_inside_env(self, req, output_dir, python_tag=None,
                     shutil.move(
                         os.path.join(temp_dir.path, wheel_name), wheel_path
                     )
+                except BaseException as exc:
+                    logger.critical('Failed to build: %s', exc)
```

Comment:
This should not be a critical log as there is nothing critical here: if pip can't build the wheel it will resort to simply installing from sdist.

Reply:
This method is only used in wheel building. If a move here fails, the build of the wheel fails. Thus why I had to unearth the exception to work out why my wheel building was failing.

Reply:
@cooperlees Normally, if pip tries to build a wheel and that fails, it falls back to installing from an sdist directly. Thus, you'd probably want to change it to:

```
logger.info('Failed to build wheel for: %s", req.name)
```

(or `req.project_name`, I don't remember)